### PR TITLE
KFSPTS-5325 KFSPTS-5508 creating a local surefire testing profilem an…

### PR DESCRIPTION
**Pull request by jhulslander**
…d excluding tests that run long locally.  Also made general updates for some tests to run locall.  This was coordinated fix between Daniela and Jay.

Daniela fixed a couple of tests, I created the new mave profile so owe can run the test suite locally. Not everything passes at this point.

I will create separate Jiras to evaluate and remove or fix as appropriate the long running tests.  I will also create Jiras to fix the tests that are failing or are throwing errors.

To run locally from the maven command is: "mvn -PfastLocalTests test -Dadditional.kfs.config.locations=/Users/jdh34/kuali/test/dev/kfs-config-unt.properties -DskipTests=false "

### CODE REVIEW

**Comment by jhulslander** [comment]
@db483 Looping you in on this PR as well.

**Comment by jhulslander** [comment]
something is awry

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
This is ready for review again, the issues I was having was needed to add edu/cornell/kfs/cu-spring-rice-krad-overrides.xml to my class path in properties file

**Comment by jhulslander** [comment]
retest this please

**Comment by bhutchinson** [comment]
Looks ok to me

File: pom.xml [hunk 1]
```diff
@@ -34,11 +62,11 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.17</version>
         <configuration>
-          	<argLine>-Xmx2G -Xms2G -XX:MaxPermSize=512m</argLine>
```
Comment (bhutchinson) [inline pom.xml]: did you want to remove the MaxPermSize setting?

**Comment by bhutchinson** [comment]
Other than my one question, this looks ok to me. I'm out of the office tomorrow, but pending the answer to that question and a successful unit test suite run, I think Daniela can merge this if it looks ok to her.

**Comment by cbower4545** [comment]
retest this please

**Comment by cbower4545** [comment]
retest this please

File: pom.xml (hunk 1 above)
Comment (jhulslander) [inline pom.xml]: When I was running locally I got notices that MaxPermSize has been deprecated (or something to that effect) and is being ignored, so I thought might as well get rid of it.  Should I put it back in?

**Comment by jhulslander** [comment]
Hold off on merging this please, I want to add a few quick fix unit tests errors

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
I put the -XX:MaxPermSize=512m param back in per pull request recommendation

**Pull request merged**